dvc/tree/base.py

```diff
@@ -48,7 +48,6 @@ class BaseTree:
     scheme = "base"
     REQUIRES = {}
     PATH_CLS = URLInfo
-    JOBS = 4 * cpu_count()
```

Comment:
Doesn't seem like there is a need to delete this. Or am I missing something?

Reply:
Well... after I understood the way how `voloptus` validator types work I see that there is no need to delete it, even more, it is breaking the logic :)